zipkin-storage/elasticsearch/src/main/java/zipkin2/elasticsearch/internal/client/HttpCall.java

```diff
@@ -158,6 +161,16 @@ public Factory(HttpClient httpClient) {
         ctx -> response.aggregateWithPooledObjects(ctx.eventLoop(), ctx.alloc()),
         // This should never be used in practice since the module runs in an Armeria server.
         response::aggregate);
+    responseFuture = responseFuture.exceptionally(t -> {
+      if (t instanceof UnprocessedRequestException) {
+        Throwable cause = t.getCause();
+        Exceptions.clearTrace(cause);
+        throw new RejectedExecutionException("Could not process request.", cause);
```

Comment:
would be nice to have the message in the exception here because this will also show up on health check potentially, and that only shows the direct message iirc. IOTW I don't think "Could not process request." is more telling than the cause.getMessage()

Reply:
I agree that the current message isn't super useful but stuttering an exception message is generally considered bad form and just makes stack traces redundant/noisy.  Is there a more verbose message we can put here that illustrates the point without being repetitive?  I'm not sure what causes an UnprocessedRequestException exactly so I can't immediately offer any suggestions.

Reply:
the reason I say to replace the message is that it is artificial that we are wrapping at all, that it makes the health check unreadable, and also it will be hard to guess the format of the upstream message in a way that doesn't change. This would be obvious with a unit test.

In other words, I think it serves us better to make things like health checks usable vs spend time on an artificial thing only needed because we aren't making an exception registry for the throttling system